releases/release-5.4.2.md

```diff
@@ -0,0 +1,115 @@
+---
+title: TiDB 5.4.2 Release Notes
+---
+
+# TiDB 5.4.2 Release Notes
+
+Release Date: xx xx, 2022
+
+TiDB version: 5.4.2
+
+## __unsorted
+
++ PingCAP/TiDB
+
+    <!--planner-->
+    - ```release-note [#34613](https://github.com/pingcap/tidb/issues/34613)
+
++ Tools
+
+    + BR
+
+        - ```release-note [#35279](https://github.com/pingcap/tidb/issues/35279)
+        - ```release-note [#34865](https://github.com/pingcap/tidb/issues/34865)
+        - ```release-note [#34956](https://github.com/pingcap/tidb/issues/34956)
+        - ```release-note [#34350](https://github.com/pingcap/tidb/issues/34350)
+        (dup: release-6.0.0-dmr.md > Bug fixes> Tools> Backup & Restore (BR))- Fix a bug that BR gets stuck when the restore operation meets some unrecoverable errors [#33200](https://github.com/pingcap/tidb/issues/33200)
+
+    + TiDB Lightning
+
+        (dup: release-6.1.0.md > Improvements> Tools> TiDB Lightning)- Optimize Scatter Region to batch mode to improve the stability of the Scatter Region process [#33618](https://github.com/pingcap/tidb/issues/33618)
+
+    + PingCAP/TiCDC
```

Comment:
`PingCAP/TiCDC` is expected?

Reply:
Thanks for the reminder. That's auto-generated by a script. I'll edit them later.